### PR TITLE
docs(install): some bugs

### PR DESCRIPTION
**Pull request by kilosonc**

### CODE REVIEW

**Review by bysph — APPROVED** [review]

File: docs/tutorials/how-to-install.md [hunk 1]
```diff
@@ -158,9 +158,9 @@ helm install my-ingress-nginx -n ingress-nginx ingress-nginx/ingress-nginx --ver
 docker ps
```
Comment (iamyeka) [inline docs/tutorials/how-to-install.md:158]: remove docker ps ?

File: docs/tutorials/how-to-install.md [hunk 2]
```diff
@@ -214,10 +214,22 @@ docker ps
 #CONTAINER ID   IMAGE                    COMMAND                  CREATED        STATUS        PORTS                                                                                                                                                                                                                                                                                                     NAMES
 #d5b8bad67208   kicbase/stable:v0.0.36   "/usr/local/bin/entr…"   21 hours ago   Up 21 hours   0.0.0.0:80->80/tcp, :::80->80/tcp, 0.0.0.0:443->443/tcp, :::443->443/tcp, 0.0.0.0:49167->22/tcp, :::49167->22/tcp, 0.0.0.0:49166->2376/tcp, :::49166->2376/tcp, 0.0.0.0:49165->5000/tcp, :::49165->5000/tcp, 0.0.0.0:49164->8443/tcp, :::49164->8443/tcp, 0.0.0.0:49163->32443/tcp, :::49163->32443/tcp   minikube
 
-docker exec -it d5b8bad67208 bash
+docker exec -it minikube bash
 
 # in container
```
Comment (iamyeka) [inline docs/tutorials/how-to-install.md:219]: if you use if this way, in container seems not right

File: docs/tutorials/how-to-install.md [hunk 3]
```diff
@@ -214,10 +214,22 @@ docker ps
...
-echo "nameserver `kubectl get service -n kube-system kube-dns -o jsonpath='{.spec.clusterIP}'`" > /etc/resolv.conf
+kubectl get service -n kube-system kube-dns -o jsonpath='{.spec.clusterIP}' | xargs -I {} docker exec minikube bash -c 'echo "nameserver {}" > /etc/resolv.conf'
+```
+
+:::
+
+:::caution
+
+If you encounter `503 Service Temporarily Unavailable` error when visiting Horizon, please try to install ingress manually.
```
Comment (iamyeka) [inline docs/tutorials/how-to-install.md:227]: Just use helm to install ingress-ngnix for consistence with Kind. Addon may run not as expected.

**Review by iamyeka — CHANGES_REQUESTED** [review]

File: docs/tutorials/how-to-install.md (hunk 1 above)
Comment (kilosonc) [reply docs/tutorials/how-to-install.md:158]: fixed

File: docs/tutorials/how-to-install.md (hunk 2 above)
Comment (kilosonc) [reply docs/tutorials/how-to-install.md:219]: fixed

**Review by iamyeka — APPROVED** [review]

**Pull request merged**